### ISSUE TITLE
how to use the fine-tuning model to batch generate text on your own test set and save it

### ISSUE THREAD

**Issue by xjy2020** [pytorch/torchtune#937]
May I ask how to use the fine-tuning model to batch generate text on your own test set and save it?

**Comment by kartikayk** [comment]
Thanks for opening this issue!

We currently don't support batch inference. But @joecummings is working on this as part of enabling batch evaluation. We should have a PR on this soon.

**Comment by xjy2020** [comment]
Thank you for your answer. I have modified the generate.py file and resolved this issue. Torchtune is a great job!

**Comment by varunsatish** [comment]
@xjy2020 would you be willing to let me take a look at your code to see how you did it?
